website/src/content/main-concepts/user-permissions.mdx

```diff
@@ -2,10 +2,201 @@
 title: User Permissions
 ---
 
-<Info>
+The Merchant Center allows configuring permissions based on a user’s team. Each teams belong to one organization. While each organization can have as numerous teams and a user can belong to multiple teams across one or different organizations.
 
-See [User Permission Concepts](https://docs.commercetools.com/merchant-center/user-permissions.html).
+More information about the [User Permission Concepts](https://docs.commercetools.com/merchant-center/user-permissions.html).
 
-</Info>
+# Types of permissions
 
-_Coming soon_
+The Merchant Center loads and provides different types of permissions to be used within a Custom Application.
+
+## General permissions
+
+General permissions are bound to a resource and grant either view or manage  access. Whenever a user has manage access the respective view access is granted implicitly.
+
+A general permission is represented as a `String` with a prefix of `View` or `Manage`, followed by one of the following resource names:
+
+- `Products`
+- `Categories`
+- `Customers`
+- `CustomerGroups`
+- `Orders`
+- `ProductDiscounts`
+- `CartDiscounts`
+- `DiscountCodes`
+- `ProjectSettings`
+- `ProductTypes`
+- `DeveloperSettings`
+
+## Action rights
+
+Action rights can be used to constrain a granted general permission. As manage access to products can be limited by not allowing publishing of products for a team.
+
+An action right is represented as an object with the shape `{ group: string, name: string }`. The `group` relates to the permission while the `name` is the action right itself. Currently the following action rights for the `group: 'products'` exist:
```

Comment:
Q: should we also include`value`?

Reply:
I wouldn't no - at least how I understand it. I think it's internal and also not exposed [ref](https://github.com/commercetools/merchant-center-application-kit/blob/master/packages/permissions/src/utils/has-permissions.ts#L12).